examples/project/callibrate_data.py

```diff
@@ -0,0 +1,7 @@
+# """Code for maintaining the data before storing it to the database"""
+# class DataCallibration():
```

Comment:
This looks to be commented out. If it's not needed lets remove it. If it is still needed please un-comment and show it's usage in one of the examples.

Reply:
Made necessary changes.
Admin can delete callibrate_data.py